skbio/stats/power.py

```diff
@@ -295,9 +307,22 @@ def subsample_power(test, samples, draw_mode='ind', alpha_pwr=0.05,
                 raise ValueError('Each vector in samples must be the same '
                                  'length in "matched" draw_mode.')
 
-    # Checks there are enough samples to subsample
-    if num_ids <= min_observations:
-        raise ValueError('There are not enough samples for subsampling.')
+    # Checks the ratio
+    num_groups = len(samples)
+    if ratio is None:
+        ratio = np.ones((num_groups))
+    ratio = np.asarray(ratio)
```

Comment:
Can you please move this casting to an `else` clause?

For the error checking that follows it, I don't mind if you move it or not. It's not necessary when ratio is None, but it's cheap/doesn't hurt to do.